src/internal/pctx/http.go

```diff
@@ -0,0 +1,9 @@
+package pctx
+
+// // ConnContextFn returns a function suitable for use as an (http.Server).ConnContext function.  It
+// // creates a new context containing pctx information and a logging span for each incoming request.
+// func ConnContextFn(ctx context.Context) func(l net.Listener) context.Context {
```

Comment:
What's going on here?

Reply:
Very early in the refactor I wasn't quite sure how I was going to get loggable contexts into the bare HTTP servers.  Ended up doing it in `log` though that will probably move to `pctx` some day.